core/src/App.html

```diff
@@ -1393,46 +1402,80 @@
   }
 
   //Fixing issues for fundamental styles
+
+  //Add padding for shellbar buttons
   :global(.fd-shellbar__button) {
-    padding-left: 12px;
-    padding-right: 12px;
+    padding-left: 0.5625rem !important;
+    padding-right: 0.5625rem !important;
   }
 
-  //Overwrites with SAP variables
-
-  :global(.fd-shellbar) {
-    border-bottom: #354a5f;
-    border-bottom: 1px solid var(--sapShell_BorderColor, #354a5f);
+  :global(.fd-menu__link) {
+    text-decoration: none; //Remove underline from links in fd-menu
+    border-radius: 0; //Remove border radius from links inside fd-menu
   }
 
-  :global(.fd-menu__item) {
-    background-color: var(--sapBackgroundColor, #fff);
-    color: var(--sapLinkColor, #51555a);
-
-    &.is-hover,
-    &:hover,
-    &.is-selected:hover {
-      color: #0854a0;
-      color: var(--sapLink_Hover_Color, #0854a0);
-      background-color: #ebf5fe;
-      background-color: var(--sapButton_Hover_Background, #ebf5fe);
+  //Make border radius for links inside fd-menu only for the first and last elements
+  :global(.fd-menu) {
+    :global(.fd-menu__list) {
+      &:first-child {
+        :global(.fd-menu__item) {
+          &:first-child {
+            :global(.fd-menu__link) {
+              border-top-left-radius: 0.25rem;
+              border-top-left-radius: var(
+                --sapElement_BorderCornerRadius,
+                0.25rem
+              );
+              border-top-right-radius: 0.25rem;
+              border-top-right-radius: var(
+                --sapElement_BorderCornerRadius,
+                0.25rem
+              );
```

Comment:
Just a tiny suggestion: since this part is the same for last and first child: 
```
@mixin luigi-menu__link--border {
            :global(.fd-menu__link) {
              border-top-left-radius: 0.25rem;
              border-top-left-radius: var(
                --sapElement_BorderCornerRadius,
                0.25rem
              );
              border-top-right-radius: 0.25rem;
              border-top-right-radius: var(
                --sapElement_BorderCornerRadius,
                0.25rem
              );
}

 :global(.fd-menu) {
    :global(.fd-menu__list) {
      &:first-child {
        :global(.fd-menu__item) {
          &:first-child {
          @include luigi-menu__link--border;
              }
          }
        }
      }


```
If the numbers change we would need to change it once

Reply:
thanks for the suggestion, it would be great to use, but they are not the same. I do `border-radius` for **top** corners of the first element, and `border-radius` for **bottom** corners of the last element.